### PR TITLE
Support Mul and Sub in padding elimination

### PR DESCRIPTION
**Pull request by guyang3532**
### Description
Support Mul and Sub in padding elimination

### CODE REVIEW

File: orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py [hunk 1]
```diff
@@ -5874,25 +5892,26 @@ def generate_inputs(batch_size, max_seq_length, vocab_size):
     else:
         assert len([node.op_type for node in training_model.graph.node if node.op_type == "ShrunkenGather"]) == 1
     gathergrad_node = [node for node in training_model.graph.node if node.op_type == "GatherGrad"][0]
-    if test_op == "Add":
+    def find_input_node_type(model, arg):
+        result = []
+        for node in model.graph.node:
+            for output in node.output:
```
Comment (pengwa) [inline orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py:5898]: ifr arg in node.output:
  result.append(node)

File: orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc [hunk 2]
```diff
@@ -292,7 +292,9 @@ void IterateSubgraphFromNode(Graph& graph,
     to_visit.pop();
     visited.insert(cur);
     if (graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "Add", {7, 13, 14}) ||
-        graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "BiasGelu", {1}, kMSDomain)) {
+        graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "BiasGelu", {1}, kMSDomain) ||
+        graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "Sub", {7, 13, 14}) ||
+        graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "Mul", {1, 6, 7, 13, 14})) {
```
[the remark this answers is not in the document]
Comment (guyang3532) [reply orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc]: done

File: orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py [hunk 3]
```diff
@@ -5834,8 +5849,8 @@ def test_other(self, input_ids):
             return output
 
         def forward(self, input_ids):
-            if test_op == "Add":
-                output = self.test_add(input_ids)
+            if test_op == "Add" or test_op == "Mul" or test_op == "Sub":
```
[the remark this answers is not in the document]
Comment (guyang3532) [reply orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py]: done

File: orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py (hunk 1 above)
Comment (guyang3532) [reply orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py]: done

**Review by pengwa — APPROVED** [review]

**Pull request merged**